### PR TITLE
feat: Add a ls-spaces command to list configured spaces

### PR DESCRIPTION
**Pull request by sblaisot**
Add a command to list configured spaces, with flags to be able to list only physical or virtual spaces

### CODE REVIEW

File: cmd/space.go [hunk 1]
```diff
@@ -47,3 +47,26 @@ var rmSpaceCmd = &cobra.Command{
...
+
+var lsSpaceCmd = &cobra.Command{
+	Use:     "ls-spaces",
+	Short:   `List spaces`,
+	Long:    `List all configured spaces`,
+	PreRunE: compose(prepareRegistry, prepareSpaces),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		var spaceList []string
+
+		if physicalFlag || (!physicalFlag && !virtualFlag) {
```
Comment (taratatach) [inline cmd/space.go:59]: This could be simplified:
```suggestion
		if physicalFlag || !virtualFlag {
```

File: cmd/space.go [hunk 2]
```diff
@@ -47,3 +47,26 @@ var rmSpaceCmd = &cobra.Command{
...
+			spaceList = config.GetSpaces()
+		}
+
+		if virtualFlag || (!physicalFlag && !virtualFlag) {
```
Comment (taratatach) [inline cmd/space.go:63]: Same here:
```suggestion
		if virtualFlag || !physicalFlag {
```

**Review by taratatach — CHANGES_REQUESTED** [review]

File: cmd/space.go (hunk 1 above)
Comment (sblaisot) [reply cmd/space.go]: right. Updated.

File: cmd/space.go (hunk 2 above)
Comment (sblaisot) [reply cmd/space.go]: Updated also

**Pull request merged**